### ISSUE TITLE
Make an instance of the elucidate annotation repository available for the team

### ISSUE THREAD

**Issue by hennie** [CLARIAH/clariah-plus#54]

**Comment by hennie** [comment]
Who would like this? And in what form: docker for local usage, instance shared online, ...?

**Comment by hsloetjes** [comment]
We would like to be able to test against and use a shared online instance (in addition to local testing that we're planning to do).

**Comment by jblom** [comment]
@hennie @hsloetjes @mmisworking I'd say it makes most sense to  host one in the shared k8s cluster intended for all the running CLARIAH services, meaning the server should be containerised. Is there already a docker image available?

**Comment by brambg** [comment]
Yes, there is a docker image available. I just haven't tried it out on a k8s cluster yet.
If someone can send me instructions on how to use the aforementioned "shared k8s cluster" I can take a stab at installing elucidate there.

**Comment by jblom** [comment]
@brambg ah nice, yes @mmisworking can give you access to the current k8s cluster. Don't hesitate to ask helping out with defining the k8s yaml configs etc.

**Comment by mmisworking** [comment]
@brambg the format in the dev cluster is <something>.dev.clariah.nl. can you deliver the <something>. at least if you want to expose 443/80 to the world for this service.

**Comment by brambg** [comment]
There is now an instance of huc-elaborate up and running on https://elucidate.dev.clariah.nl/

**Comment by jblom** [comment]
@brambg good to see you got elucidate running (in the k8s cluster I presume?)

**Comment by brambg** [comment]
Yes, @jblom , it's in the clariah k8s dev cluster
